package USER_RESPAWN;

message CONTENTS
{
	optional int32 id = 1;
	optional int32 x = 2;
	optional int32 y = 3;
}